omero/figure/index.html

```diff
@@ -22,7 +22,7 @@ <h1 class="hero-main-header"><img src="{{ site.baseurl }}/img/logos/omero-figure
                     <h4>Create quality figures<br>from your OMERO images</h4>
                     <br>
                     <a href="https://pypi.python.org/pypi/omero-figure/" class="large btn-red button sites-button" style="text-shadow: none;">Install OMERO.<span class="lowercase">f</span>igure</a>
-                    <a href="{{ site.baseurl }}/2019/10/28/figure-4-1-0.html" class="hero-link-dark" style="text-shadow: none;">Read the Release Announcement</a>
+                    <a href="{{ site.baseurl }}/2020/05/15/figure-4-3-0.html" class="hero-link-dark" style="text-shadow: none;">Read the Release Announcement</a>
```

Comment:
Are we planning to ship a software release today (i.e. on Friday) or better to wait till Monday?

Reply:
I think we are OK to release figure today. It's a minor release and is unlikely to get lots of installs and bugs reported over the weekend.
Would be good to get it out of the way.

Reply:
I think we are fine to push it today, considering everything we have on our plate at the moment